core/src/main/java/feign/FeignException.java

```diff
@@ -57,7 +57,11 @@ public int status() {
   }
 
   public String contentUTF8() {
-    return new String(content, UTF_8);
+    if (content != null) {
+      return new String(content, UTF_8);
+    } else {
+      return null;
```

Comment:
It may be better to return an empty string here instead of `null`.  Unless others look into this code, we may be pushing the null pointer higher.  Thoughts?

Reply:
Good point. I thought one might want to differentiate whether the content was empty (`""`) or `null`. Eventually, the method could even return an `Optional<String>` with or without the content. I see your point though, and assume it's more convenient to simply return an empty String :)